### DISCUSSION TITLE
v1.0.0-rc3

### DISCUSSION THREAD

**Discussion by Mrs4s** [Mrs4s/go-cqhttp#1548] [General]
## Changelog
此版本恢复了滑条验证码的支持, 如果无法通过扫码登录可尝试更新

### 新增
859f40d 消息资源支持了 `base16384 ` 编码.  @fumiama 
23d594b 新增私聊文件API `upload_private_file`. @wdvxdr1123 
2a0baba 重新引入了滑条验证码的支持, 以应对二维码风险网络的问题. @Mrs4s 

### 修复
7d97216 修复了data文件夹的权限问题. @LambdaYH 
7e24f8b 修复了处理事件出现错误的日志消息可能会错误的混入上报信息的问题. @Mrs4s 
7349fd4 修复了频道发送已存在图片时可能会崩溃的问题. @Mrs4s 
b013f66 修复了上传图片时可能会崩溃的问题. @Mrs4s 
babf35e 修复了群消息事件会存在两个 `message_type` 字段的问题. @wdvxdr1123 
修复了无法登录刚解封的账号的问题. @Mrs4s 
修复了无法正常处理滑条验证码的问题. @Mrs4s ( 锅 @fumiama )
修复了无法上传群文件的问题. @wdvxdr1123 

### 协议更新
`Android Phone` 协议更新到了 `8.8.95` 版本. @wdvxdr1123 

<hr /><em>This discussion was created from the release <a href='https://github.com/Mrs4s/go-cqhttp/releases/tag/v1.0.0-rc3'>v1.0.0-rc3</a>.</em>

**Comment by MistEO** [comment]
太棒了，服务器之前一直被 tx 限制，现在终于能登上新号了。赞美大佬们！

**Comment by fumiama** [comment]
我直接谢罪
![~ 427J0 JKI%{DHO3JANB(C](https://user-images.githubusercontent.com/41315874/175540480-ee341a5c-bb1a-4d6f-8ea1-b6a5f9e2451d.jpg)

**Comment by vaecou** [comment]
谢谢大佬们新增私信文件的api！

**Comment by xiwangly2** [comment]
太棒了

**Comment by 0x24a** [comment]
卧槽，太强了

**Comment by xiazhi0** [comment]
文档快点跟进一下嘛❤🧡💛

**Comment by LDCRlbc** [comment]
想问一下这个需要下载哪些文件呀

**Reply by Starrr-sea** [reply]
这要看是什么系统，下载对应的文件运行即可，但是相应的数据相应是需要您自行编写的

**Comment by FUXINWF** [comment]
Windows_amd64和_386有什么区别啊

**Reply by Ink-33** [reply]
64位和32位，请火速使用百度搜索

**Comment by Pessimisticc** [comment]
为什么我Centos，http上报给PHP，请求体全部为空呀😭

**Comment by 2536** [comment]
v1.0.0-rc3没有了go-cqhttp_linux_arm64.tar.gz吗

**Reply by Starrr-sea** [reply]
linux系统已经有安装包了，rpm和deb文件是Linux系统的可执行的安装文件

**Comment by Starrr-sea** [comment]
有个问题，QQ群聊的邀请新人会出现无法获取到邀请人的qq，该值变为了0，在最新版的rc3里依旧有这个问题，请问有什么解决方案吗
系统返回的值如下，其中operator_id的邀请人字段为0
```
{ 'post_type': 'notice', 
  'notice_type': 'group_increase',
  'time': 1664247037, 
  'self_id': 3276394070, 
  'sub_type': 'approve',
  'group_id': 953026557,
  'operator_id': 0,
  'user_id': 3411808390}
```

**Reply by HJWqa** [reply]
建议提交issue

**Comment by DDG667** [comment]


**Reply by Ink-33** [reply]
你的问题。
RDFM

**Reply by DDG667** [reply]
> 你的问题。
> RDFM

谢谢，已经解决了，nb创建时没选好

**Comment by sgj123456** [comment]
Mac下载哪个

**Comment by qq625324542** [comment]
linux .rpm的可不可以出个使用文档，不会用这个。

**Reply by Ink-33** [reply]
`sudo dnf install xxx.rpm`

**Comment by qq625324542** [comment]
您好，谢谢回复，后面怎么执行呢

坏小孩
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "Mrs4s/go-cqhttp"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年11月1日(星期二) 上午10:35
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [Mrs4s/go-cqhttp] v1.0.0-rc3 (Discussion #1548)





 
sudo dnf install xxx.rpm

**Reply by Ink-33** [reply]
直接输`go-cqhttp`运行

**Comment by qq625324542** [comment]
主要是没找到生成的配置文件在哪个位置


坏小孩
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "Mrs4s/go-cqhttp"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年11月1日(星期二) 上午10:40
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [Mrs4s/go-cqhttp] v1.0.0-rc3 (Discussion #1548)





 
直接输go-cqhttp运行

**Comment by zmc1009** [comment]
为什么他一直说配置文件与缓存文件不相同，有什么解决办法嘛

**Reply by x7z0** [reply]
配置文件是旧版本生成的吧，重新创建就行
